### DISCUSSION TITLE
Error: nippy jar error: (os error 5)

### DISCUSSION THREAD

**Discussion by MiraWells** [paradigmxyz/reth#9954] [General]
Error: nippy jar error: 拒绝访问。 (os error 5)
![351521955-5db91626-a66b-4eba-a9f3-b35ac22f78df](https://github.com/user-attachments/assets/f6ea54c2-09e3-4cd5-8abb-0398d8d72c90)

**Comment by shekhirin** [comment]
Please check permissions on the `.\\db` directory. OS error 5 means "Access Denied".

**Reply by MiraWells** [reply]
I've carefully checked, and it's definitely not an operating system access permission issue. This problem has persisted for over a month. Whether it's creating a database in an empty folder or using files downloaded from a snapshot, the error always occurs. Could you please check this issue on a real Windows system? I would really appreciate it.
